src/richie/apps/courses/templates/courses/cms/fragment_course_run.html

```diff
@@ -18,7 +18,7 @@ <h3>{{ run.title|capfirst }}</h3>
     {% if run|has_connected_lms %}
         <div
             class="richie-react richie-react--course-run-enrollment"
-            data-props='{"courseRunId": {{ run.id }}}'
+            data-props='{"courseRun": {"id": {{ run.id }}, "resource_link": "{{ run.resource_link }}", "priority": {{ run.state.priority }}}}'
```

Comment:
We're reaching the limit of what should be done in the template (cf the test as well...). Maybe make a small filter or tag like it was done for "has_connected_lms"?